README.md

```diff
@@ -1,2 +1,46 @@
 # container-ipam-tool
 Configuration tool for any container environments
```

Comment:
Configuration tool for any container environments which uses Infoblox Appliance for their IPAM. This tools pre configure Infoblox Appliance before using it from container environment. 

Extensible Attributes in Infoblox Appliance help to manage IPAM objects effectively for any cloud platform. So few EA are needed in Infoblox Appliance for container environment this tools helps to create those EAs